### PR TITLE
fix: accessing stp_strings[49] (is out of bounds)

### PR DESCRIPTION
**Pull request by gertius1**
change: reactivated windows scribble strip
change: got rid of warning by initializing stp_colors

### CODE REVIEW

File: source/CSurf_US2400_stripoverlay.cpp [hunk 1]
```diff
@@ -52,22 +52,26 @@ CSurf_US2400_stripoverlay::CSurf_US2400_stripoverlay() {
...
-	// stp_class.hInstance = g_hInst;
-	// stp_class.hIcon = LoadIcon(NULL, IDI_APPLICATION);
-	// stp_class.hCursor = LoadCursor(NULL, IDC_ARROW);
-	// stp_class.hbrBackground = CreateSolidBrush(RGB(60, 60, 60));
-	// stp_class.lpszMenuName = NULL;
-	// stp_class.lpszClassName = "stp";
-	// stp_class.hIconSm = LoadIcon(NULL, IDI_APPLICATION);
-
-	// RegisterClassEx(&stp_class);
+	WNDCLASSEX stp_class;
```
Comment (Ruxton) [inline source/CSurf_US2400_stripoverlay.cpp:62]: Wrap 60-74 in some conditionals so it only happens in win32 with `#ifdef _WIN32` `#endif`

WNDCLASSEX doesn't exist on other platforms.

File: source/csurf_us2400_helpoverlay.cpp [hunk 2]
```diff
@@ -38,20 +38,21 @@ CSurf_US2400_helpoverlay::CSurf_US2400_helpoverlay() {
...
-	// hlp_class.hInstance = g_hInst;
-	// hlp_class.hIcon = LoadIcon(NULL, IDI_APPLICATION);
-	// hlp_class.hCursor = LoadCursor(NULL, IDC_ARROW);
-	// hlp_class.hbrBackground = CreateSolidBrush(RGB(60, 60, 60));
-	// hlp_class.lpszMenuName = NULL;
-	// hlp_class.lpszClassName = "hlp";
-	// hlp_class.hIconSm = LoadIcon(NULL, IDI_APPLICATION);
-
-	// RegisterClassEx(&hlp_class);
+	WNDCLASSEX hlp_class;
```
Comment (Ruxton) [inline source/csurf_us2400_helpoverlay.cpp:42]: Wrap 41-55 in some conditionals so it only happens in win32 with `#ifdef _WIN32` `#endif`

WNDCLASSEX doesn't exist on other platforms.

**Review by Ruxton — CHANGES_REQUESTED** [review]
Thanks a lot for the PR, if you could just put some build conditionals around WNDCLASSEX stuff I'll merge it up.

**Comment by gertius1** [comment]
Hi Ruxton, thanks for your comment, I just added the build conditionals.
Makes sense, I didn´t have multiplatform on my radar but will take it into account in the future.
Thanks!

**Review by Ruxton — APPROVED** [review]

**Pull request merged**